project/Build.scala

```diff
@@ -8,8 +8,8 @@ object ScalajsReactComponents extends Build {
 
   val Scala211 = "2.11.7"
 
-  val scalajsReactVersion = "0.9.2"
-  val scalaCSSVersion = "0.3.0"
+  val scalajsReactVersion = "0.10.0a-SNAPSHOT"
```

Comment:
To build you need to `publishLocal` artifacts with these version numbers from newest code in git. need scalacss too because of linking issues